docs/build/interoperate-qiskit-qasm2.mdx

```diff
@@ -68,7 +51,7 @@ By default, this parser is a little bit more relaxed than the official specifica
 
 ### Import an OpenQASM 2 program as a string
 
-Use `loads()` to import an OpenQASM 2 program as a string into a QuantumCircuit:
+Use `qasm2.loads()` to import an OpenQASM 2 program as a string into a QuantumCircuit:
```

Comment:
I think it would be better to split up these examples with their corresponding explanation paragraphs above, rather than shoving all the code snippets at the end of the page. So the "import a qasm2 program as a string" example should go higher under the 'Import an OpenQASM 2 program into Qiskit" section at the top, and the same for each of the other code examples here

Reply:
Addressed [here](https://github.com/Qiskit/documentation/pull/239/commits/517a6804c0730c35aa0f8eeacf1e0885127394a3) -- lmk if things didn't line up right. Strict mode doesn't seem to have an example so it's by its lonesome at the end.